### DISCUSSION TITLE
Budgeting for OSS sponsorships/funding

### DISCUSSION THREAD

**Discussion by mercyorangi** [github/github-ospo#27] [General]
Hi team,

Thanks for open-sourcing your OSPO sauce! I wonder if there are any best practices folks can share on how to scope out budget for OSS funding; what are the factors you consider to land on a ball-park range for budget allocation?

**Comment by zkoppert** [comment]
I'm working on putting something together on OSS funding. In the meantime, you can check out the following:

- https://blog.sentry.io/2021/10/21/we-just-gave-154-999-dollars-and-89-cents-to-open-source-maintainers
- https://github.com/microsoft/foss-fund
- https://engineering.salesforce.com/giving-back-through-our-foss-fund-38754167c00d
- https://gratipay.news/your-company-should-probably-pay-2000-per-person-for-open-source-9205443e209d

**Reply by mercyorangi** [reply]
These are great resources, thanks @zkoppert ! Seems Microsoft's FOSS Fund is in addition to supporting other OS foundations and activities, would be interesting to see what best practices they have on budget allocation for this too. I look forward to seeing your analysis when ready :)
